SwiftDeck/SwiftDeck/Communication/Messages/ConfigurationMessage.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ConfigurationMessage.swift
+//  SwiftDeck
+//
+//  Created by Ilija Puaca on 22/1/20.
+//  Copyright © 2020 Unfolded. All rights reserved.
+//
+
+/// Message type that's used to relay view configuration information.
+final class ConfigurationMessage: Message {
```

Comment:
So these messages should eventually align with the transport protocol ? https://docs.google.com/presentation/d/1fzSHJxstSfNQe8VCtLLZvQ8D3OuJAfS7_LqPg0QPPvc